realm-annotations-processor/src/main/java/io/realm/processor/RealmJsonTypeHelper.java

```diff
@@ -47,48 +48,53 @@
         JAVA_TO_JSON_TYPES.put("java.lang.String", new SimpleTypeConverter("String", "String"));
         JAVA_TO_JSON_TYPES.put("java.util.Date", new JsonToRealmTypeConverter() {
             @Override
-            public void emitTypeConversion(String setter, String fieldName, String fieldType, JavaWriter writer) throws IOException {
+            public void emitTypeConversion(String setter, String fieldName, String fieldType, JavaWriter writer)
+                    throws IOException {
                 writer
-                    .beginControlFlow("if (!json.isNull(\"%s\"))", fieldName)
+                        .beginControlFlow("if (!json.isNull(\"%s\"))", fieldName)
```

Comment:
You can tell that to checkstyle, but not to AS afaik, which is the one who reformatted this